### ISSUE TITLE
Not following system setting for light/dark mide

### ISSUE THREAD

**Issue by genebean** [statichunt/iwriter-hugo#3]
I’ve just set this up at https://volunteertechnologist.netlify.app/ and it doesn’t switch to dark mode automatically on my iPhone or my computer. Based on the code I’ve seen, the setting below from `params.toml` should be causing it to automatically change. 

```toml
# theme-mode
theme_switcher = true
theme_default = "system" # available options [light/dark/system]
```

My code is at https://github.com/genebean/hugo-volunteertechnologist.com

**Comment by tfsojon** [comment]
Hey @genebean, I’m sorry to hear you’re having trouble with the theme.
Problem solved now cdf62ccc01016887c0bf1d4742e03ccb4b81930f

Good Luck

**Comment by genebean** [comment]
Thank you!
